.github/workflows/awsDeployment.yaml

```diff
@@ -68,13 +68,15 @@ jobs:
           ENABLE_COGNITO: false
           BUCKET: phdi-playground-tfstate-${{ steps.set-environment.outputs.tf_env }}
           DYNAMODB_TABLE: phdi-playground-tfstate-lock-${{ steps.set-environment.outputs.tf_env }}
+          DB_PASSWORD: ${{ secrets.RDS_PASSWORD }}
```

Comment:
Is this now that you're generating a password?

Reply:
Aww, you are right!  This is no longer necessary with the `random_string` which now sets the db_password.  Great catch!